bundles/org.openhab.core.automation/src/main/java/org/openhab/core/automation/internal/module/handler/CompareConditionHandler.java

```diff
@@ -94,7 +93,7 @@ public boolean isSatisfied(Map<String, Object> context) {
                     case "GT":
                     case ">":
                         // Greater
-                        if (toCompare == null || rightValue == null) {
+                        if (toCompare == null) {
```

Comment:
Because of redundancy. A `null` check has been performed previously in [line 66](https://github.com/openhab/openhab-core/pull/915/files/0f7cbdf6bd150086e00c3044e22c1a12290c75e2#diff-f93d2556fc4f32e58fa867e01501accbL66).